proposals/reduce_spatial.json

```diff
@@ -0,0 +1,79 @@
+{
+    "id": "reduce_spatial",
+    "summary": "Reduce spatial dimensions 'x' and 'y",
+    "description": "Applies a unary reducer to a data cube by collapsing all the pixel values along the horizontal spatial dimensions (i.e. axes `x` and `y`) into an output value computed by the reducer. The horizontal spatial dimensions are dropped.\n\nAn aggregation over certain spatial areas can be computed with the process ``aggregate_spatial()``.\n\nThis process passes a list of values to the reducer. The list of values has an undefined order, therefore processes such as ``last()`` and ``first()`` that depend on the order of the values will lead to unpredictable results.",
```

Comment:
Just wondering: is "unary reducer" valid terminology? To me "unary" means operating on a single entity, and "reducing" is about converting a sequence of entities to a single entity. They don't seem to fit well together.

I guess we added "unary" to make the distinction with the "binary" variants, but now that we dropped these in #258, we might remove "unary" from these descriptions as well?

(feel free to address in separate issue/PR)